bodhi-messages/bodhi/messages/schemas/update.py

```diff
@@ -208,6 +220,23 @@ def summary(self) -> str:
             f"{self.update.user.name}'s {truncate(' '.join([b.nvr for b in self.update.builds]))} "
             f"bodhi update completed push to {self.update.status}")
 
+    def __str__(self) -> str:
+        """
+        Return a human-readable representation of this message.
+
+        This should provide a detailed representation of the message, much like the body
+        of an email.
+
+        Returns:
+            A human readable representation of this message.
+        """
+        new_line = "\n"
+        return (
+            f"{self.update.user.name}'s Bodhi update {self.update.alias}"
+            f"completed push to {self.update.status}\n"
+            f"Builds:\n"
+            f"{new_line.join([b.nvr for b in self.update.builds])} ")
```

Comment:
Is there any limit to a message length? What will happen with updates composed by hundreds of builds?

Reply:
I don't think there is -- @abompard might be able to weigh in on this though.

the spec states that the __str__ is supposed to be like the body of an email (with the summary being the subject) -- so IMHO, its appropriate to list everything here in the __str__